h2o-r/h2o-package/R/communication.R

```diff
@@ -40,10 +40,14 @@
     sprintf("%s://%s:%s/%s/%s", scheme, conn@ip, as.character(conn@port), h2oRestApiVersion, urlSuffix)
 }
 
-.h2o.doRawREST <- function(conn, h2oRestApiVersion, urlSuffix, parms, method, fileUploadInfo, ...) {
+.h2o.doRawREST <- function(conn, h2oRestApiVersion, urlSuffix, parms, method, fileUploadInfo, binary=FALSE, ...) {
```

Comment:
Every call to MOJO/genmodel download sets `binary=TRUE`. Unit tests for these cover that. If binary data was not sent back you would get a lot of binary text back and it would fail since the REST call would not know what to do with the payload.